tests/helpers.py

```diff
@@ -62,13 +62,13 @@ def ip_to_device_id_map(device_group, devauth=devauth):
         return ip_to_device_id
 
     @staticmethod
-    def _check_log_for_message(device, service, message, since=None):
+    def _check_log_for_message(device, message, since=None):
```

Comment:
S: Use a default argument for the service instead?

Reply:
I could have, indeed.

But after we moved the log message to `mender-auth`, and having both methods really about authentication, I thought it would make sense to hardcode `mender-authd`. Only if we extend it to grep for other general purpose messages in `mender-update` would make sense to parametrize again.